static/app/gettingStartedDocs/javascript/javascript.tsx

```diff
@@ -112,8 +146,179 @@ const getInstallConfig = () => [
+      strong: <strong />,
+    }),
+  install: params => [
+    {
+      type: StepType.INSTALL,
+      description: t('Add this script tag to the top of the page:'),
+      configurations: [
+        {
+          language: 'html',
+          code: [
+            {
+              label: 'HTML',
+              value: 'html',
+              language: 'html',
+              code: `
+<script
+  src="${params.dsn.cdn}"
+  crossorigin="anonymous"
+></script>`,
+            },
+          ],
+        },
+      ],
+    },
+  ],
+  configure: params => [
+    {
+      type: StepType.CONFIGURE,
+      description: t(
+        "Initialize Sentry as early as possible in your application's lifecycle."
+      ),
+      isOptional: true,
+      configurations: [
+        {
+          language: 'html',
+          code: [
+            {
+              label: 'HTML',
+              value: 'html',
+              language: 'html',
+              code: `
+<script>
+  Sentry.onLoad(function() {
+    Sentry.init({${
+      !(params.isPerformanceSelected || params.isReplaySelected)
+        ? `
+        // You can add any additional configuration here`
+        : ''
+    }${
+      params.isPerformanceSelected
+        ? `
+        // Tracing
+        tracesSampleRate: 1.0, // Capture 100% of the transactions`
+        : ''
+    }${
+      params.isReplaySelected
+        ? `
+        // Session Replay
+        replaysSessionSampleRate: 0.1, // This sets the sample rate at 10%. You may want to change it to 100% while in development and then sample at a lower rate in production.
+        replaysOnErrorSampleRate: 1.0, // If you're not already sampling the entire session, change the sample rate to 100% when sampling sessions where errors occur.`
+        : ''
+    }
+      });
+  });
+</script>`,
+            },
+          ],
+        },
+      ],
+      onOptionalToggleClick: showOptionalConfig => {
+        if (showOptionalConfig) {
+          trackAnalytics('onboarding.js_loader_npm_docs_optional_shown', {
+            organization: params.organization,
+            platform: params.platformKey,
+            project_id: params.projectId,
+          });
+        }
+      },
+    },
+  ],
+  verify: getVerifyConfig,
+  nextSteps: () => [
+    {
+      id: 'performance-monitoring',
+      name: t('Tracing'),
+      description: t(
+        'Track down transactions to connect the dots between 10-second page loads and poor-performing API calls or slow database queries.'
+      ),
+      link: 'https://docs.sentry.io/platforms/javascript/tracing/',
+    },
+    {
+      id: 'session-replay',
+      name: t('Session Replay'),
+      description: t(
+        'Get to the root cause of an error or latency issue faster by seeing all the technical details related to that issue in one visual replay on your web application.'
+      ),
+      link: 'https://docs.sentry.io/platforms/javascript/session-replay/',
```

Comment:
In the past, we only displayed the next step for 'tracing' if this product was not selected. However, we're no longer following this approach across many platforms. Now, we can simplify the process by displaying all available options. I would even suggest unifying the next steps for both the loader and package manager. 

Feedback welcome.

Reply:
I think some next steps only make sense if the product was not selected.

I would be curious, in general, how often these are clicked. Would it be possible to add some instrumentation in a follow-up PR? Then we can decide if these are actually useful or just a distraction at that point in time.